llvm/docs/PGOProfileFormat.rst

```diff
@@ -0,0 +1,395 @@
+===================================
```

Comment:
Perhaps name this ProfileFormat.rst, since so much is shared with pure code coverage applications. It's fine that the doc currently focuses on PGO.

Reply:
> Perhaps name this ProfileFormat.rst, since so much is shared with pure code coverage applications.

Ack. I wonder if we want to use `InstrumentationProfileFormat.rst` since SamplePGO uses different format.